.github/workflows/build-dist.yml

```diff
@@ -0,0 +1,141 @@
+name: build-dist
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: Used by electron-publish\src\gitHubPublisher.ts
+        required: false
+      CSC_LINK:
+        description: Used to codesign binaries
+        required: false
+      CSC_KEY_PASSWORD:
+        description: Used to codesign binaries
+        required: false
+    inputs:
+      os:
+        required: true
+        type: string
+      node:
+        required: false
+        type: string
+        default: ${{ vars.DEFAULT_NODE_VERSION }}
+      max_build_attempts:
+        required: false
+        type: number
+        default: 5
+      cache_dir:
+        required: false
+        type: string
+        default: dist
+      npm_network_timeout:
+        required: false
+        type: number
+        default: 120000
+
+env:
+  # Disable console advertisements during install phase
+  ADBLOCK: true
+
+jobs:
+  build:
+    name: Build signed artifacts on ${{ inputs.os }}
+    runs-on: ${{ inputs.os }}
+
+    strategy:
+      fail-fast: false
+
+    concurrency:
+      group: '${{ github.workflow }}-${{ inputs.os }}-${{ inputs.node }}-${{ github.head_ref || github.ref_name }}'
+      cancel-in-progress: true
+
+    env:
+      USE_HARD_LINKS: false
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Restore build cache
+        uses: actions/download-artifact@v3
+        with:
+          name: compile-${{ inputs.os }}-${{ inputs.node }}
+          path: ${{ inputs.cache_dir }}
+
+      - name: Install Node.js, NPM and Yarn
+        if: steps.dist_cache.outputs.cache-hit != 'true'
```

Comment:
There is no dist_cache step anymore, right?

Reply:
You're absolutely right.  Removing all `cache-hit` references...